client/package.json

```diff
@@ -27,9 +28,11 @@
     "react-dom": "18.2.0",
     "react-native": "0.72.4",
     "react-native-gesture-handler": "~2.12.0",
+    "react-native-maps": "1.7.1",
     "react-native-safe-area-context": "4.6.3",
     "react-native-screens": "~3.22.0",
-    "react-native-web": "~0.19.6"
+    "react-native-web": "~0.19.6",
+    "zustand": "^4.4.1"
```

Comment:
What is this?

Reply:
Are you talking about Zustand? if so it is a state management package like redux but much morte easier to under stand and maintenance, and much less code to write.